### DISCUSSION TITLE
Log level switch

### DISCUSSION THREAD

**Discussion by yuriy-millen** [FantasticFiasco/serilog-sinks-http#308] [Ideas]
I was wondering if there was any obsticles that prevented log lever switch from being included in the implementation?

**Comment by FantasticFiasco** [comment]
Not that I know of. I can't recall why it wasn't part of the initial sink implementation, don't think there are any good reasons that would prevent an implementation.

**Reply by yuriy-millen** [reply]
It looks trivial to add. Raised a PR for this: https://github.com/FantasticFiasco/serilog-sinks-http/pull/309

**Reply by FantasticFiasco** [reply]
I've created a pre-release, and it's published [here](https://www.nuget.org/packages/Serilog.Sinks.Http/9.0.0-beta.1). It'll stay as a pre-release for a while due to being a breaking change, but will eventually be released in an official version. Thanks for providing this functionality!

**Reply by yuriy-millen** [reply]
Amazing! Thank you for your help pushing the change in and this great project in general!
